### PR TITLE
feat: language selector

### PR DESCRIPTION
**Pull request by h8570rg**
## Summary
- Change default language from 'ja' to 'en'
- language selector

https://github.com/cam-inc/viron/assets/54166240/31ab05ec-3ade-4ca3-8c5a-89112c96062f

## Checklist
- [x] [changeset file(s)](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) included

### CODE REVIEW

File: packages/app/src/locales/en/common.json [hunk 1]
```diff
@@ -7,5 +7,9 @@
   "startButtonLabel": "Get started",
   "documentation": "Documentation",
   "releaseNotes": "Release Notes",
-  "help": "Help"
+  "help": "Help",
+  "language": {
+    "en": "English",
+    "ja": "日本語"
```
Comment (h8570rg) [inline packages/app/src/locales/en/common.json:13]: Just in case.

File: packages/app/src/hooks/navigation.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { useI18next } from 'gatsby-plugin-react-i18next';
+
+export const useNavigation = () => {
```
Comment (h8570rg) [inline packages/app/src/hooks/navigation.ts:3]: wrap navigate

File: packages/app/src/constants/index.ts [hunk 3]
```diff
@@ -8,14 +8,14 @@ export const URL = {
   GITHUB: 'https://github.com/cam-inc/viron/',
   TWITTER: 'https://twitter.com/TODO',
 } as const;
-export type Url = typeof URL[keyof typeof URL];
+export type Url = (typeof URL)[keyof typeof URL];
```
Comment (h8570rg) [inline packages/app/src/constants/index.ts:11]: eslint diff

File: packages/app/.eslintrc.js
Comment (nonoakij) [inline packages/app/.eslintrc.js:1]: :+1:

File: packages/app/src/components/contact/index.tsx [hunk 4]
```diff
@@ -22,11 +22,7 @@ const _Contact: React.FC<Props> = ({ on, className = '', data }) => {
   }, [data]);
 
   return (
-    <Link
-      className={classnames('group focus:outline-none', className)}
-      on={on}
-      to={to}
-    >
+    <Link className={classnames('group focus:outline-none', className)} to={to}>
```
Comment (nonoakij) [inline packages/app/src/components/contact/index.tsx:25]: on が消えた？

Comment (h8570rg) [reply packages/app/src/components/contact/index.tsx:25]: onはLinkで使ってないので必要ないです！

**Review by nonoakij — APPROVED** [review]

**Review by nonoakij — APPROVED** [review]

**Review by nonoakij — APPROVED** [review]

**Pull request merged**